code/modules/mob/living/living_powers.dm

```diff
@@ -15,3 +15,12 @@
 		layer = HIDING_LAYER //Just above cables with their 2.44
 		plane = OBJ_PLANE
 		to_chat(src,"<span class='notice'>You are now hiding.</span>")
+
+/mob/living/proc/toggle_selfsurgery()
+	set name = "Allow Self Surgery"
+	set desc = "Toggles the 'safeties' on self-surgery, allowing you to do so."
+	set category = "Object"
+
+	allow_self_surgery = !allow_self_surgery
+
+	to_chat(usr, "<span class='notice'>You will [allow_self_surgery ? "now" : "no longer"] attempt to operate upon yourself.</span>")
```

Comment:
Imo, I think this should have an admin-notify thing just in case someone powergames it or something. Which is of course at admins' discretion for actions taken against the said odd case

Reply:
I'd be more worried about someone spamming that for the sake of spamming adminlogs, in that case. To be clear, the change was made as a result of staff concerns over it being too easy to accidentally start doing surgery on yourself, where you happen to be stood still for 5 seconds and then misclick yourself with a surgery tool (Which notably includes _ghetto_ surgery tools, which are actually pretty common). Suddenly, what was originally a bruise and maybe a cut has turned into an open surgery incision that could become septic. Especially concerning for new players that like to click on things with other things just to see what will happen, who may then be surprised that they're doing surgery and try to move to cancel it, causing the surgery _fail_ step, and confusing them greatly.

Mostly, it's an issue of this not really being a precedented behaviour, where you have to stand still _before_ doing something for it to work (Lots of things where you have to stand still _while_ you do something though). Making it so you toggle it on forces players to actively choose to start doing it, instead of being able to accidentally start doing it without realizing that they could.

Reply:
I meant to answer sooner after I got notification of this response this morning but here I am. The concern about someone being a troll enough to blatantly toggle this off/on to spam adminlogs is in itself something for an admin to handle, too, imo. But I digress and have a mindset forged from playing on an HRP server where it's more respectful to let the doctors to the work they sign up for and not "I know the mechanics, therefore I am a doctor/engineer/scientist Mary Sue."